grpclb/src/main/java/io/grpc/grpclb/CachedSubchannelPool.java

```diff
@@ -87,8 +88,7 @@ public void run() {
     return subchannel;
```

Comment:
Do you need to call `handleSubchannelState()` in `run()` as well?

Reply:
we don't need to do that because the state remains same.